src/App.jsx

```diff
@@ -14,45 +17,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const SORT_FIELD = {
+  ALPH: 'alphabetical',
+  LENGTH: 'length',
+};
+
+function visibleGoods(goods, sortField, sortReverse) {
```

Comment:
Consider renaming the function 'visibleGoods' to a more descriptive name, like 'getSortedGoods'